src/Network/Network/Cortex/ExpressRouteGateway/SetAzureRmExpressRouteGatewayCommand.cs

```diff
@@ -73,11 +73,13 @@ public class UpdateAzureRmExpressRouteGatewayCommand : ExpressRouteGatewayBaseCm
         [Parameter(
             Mandatory = false,
             HelpMessage = "Min for the scale units for this ExpressRouteGateway.")]
+        [ValidateRange(1, 100)]
```

Comment:
this is a breaking change

Reply:
We are already validating bounds in NRP, so this will just fail the call earlier.